release.sh

```diff
@@ -0,0 +1,330 @@
+#!/bin/bash
+red=`tput setaf 1`
+green=`tput setaf 2`
+yellow=`tput setaf 3`
+blue=`tput setaf 4`
+reset=`tput sgr0`
+
+#--------------------------------
+# Functions
+#--------------------------------
+function show_help {
+cat <<EOF
+
+Usage:
+  release.sh [-hvs] [-c <command>] [version]
+
+Arguments:
+  release      - Creates a GitHub released based on existing version tag
+  bump_version - Creates a new version tag
+  version      - The target version (ex: 3.2.1)
+
+Options:
+  -h            Display this help message
+  -v            Verbose mode
+  -s            Simulate only (do not release)
+  -c <command>  Runs only a single command. Possible commands are:
+                  - bump_version: generate a new version tag
+                  - release: release a new version
+
+EOF
+}
+
+function invalid_usage {
+  printf $red
+  echo $@
+  echo "Aborting..."
```

Comment:
The script is really fast, so no need for the `...`
I guess `"Execution Aborted"` is enough.